### PR TITLE
New init possibilities for dialog

### PR DESCRIPTION
**Pull request by Akswii**
New init baby 💯 🔥

### CODE REVIEW

File: src/px-script/main/dialog/index.js [hunk 1]
```diff
@@ -13,6 +13,11 @@ class Dialog {
         this.id = el.id;
         this.closeIcon = el.querySelector(SELECTORS.CLOSEICON);
         this.isOpen = el.classList.contains("d-flex");
+        this.openBtns = document.querySelectorAll(`[data-dialog-open=${this.id}]`);
+
+        if (!this.openBtns.length) {
+            console.warn(`No corresponding buttons found for dialog with id ${this.id}`);
```
Comment (erikhallaraaker) [inline src/px-script/main/dialog/index.js]:
```suggestion
            console.warn(`No corresponding open-button found for dialog with id ${this.id}`);
```

File: src/px-script/main/utils/index.js [hunk 2]
```diff
@@ -30,8 +30,28 @@ export const handleScrollbar = () => {
...
+    const pushElement = id => {
+        const val = document.querySelector(`#${id}`);
+
+        val ? elements.push(val) : console.warn(`No ${elementName} found with ID: ${id}`);
+    };
+
+    if (typeof ids === "string") {
+        pushElement(ids);
+    } else if (Array.isArray(ids)) {
+        ids.forEach(elem => pushElement(elem));
```
Comment (erikhallaraaker) [inline src/px-script/main/utils/index.js]:
```suggestion
        ids.forEach(elem => typeof elem === "string" ? pushElement(elem) : console.warn(`Skipping init value ${elem}, must be of type string.`);
```

File: src/App/Documentation/components/Dialog/index.js [hunk 3]
```diff
@@ -58,7 +58,7 @@ const UsageWithJavascript = () => (
 
 class Dialog extends Component {
     componentDidMount () {
-        dialog.init();
+        // dialog.init();
```
Comment (erikhallaraaker) [inline src/App/Documentation/components/Dialog/index.js]: No more init?

File: src/px-script/main/utils/index.js [hunk 4]
```diff
@@ -30,8 +30,28 @@ export const handleScrollbar = () => {
     }
 };
 
+export const findElemsWithId = (ids, elementName) => {
+    const elements = [];
+    const pushElement = id => {
+        const val = document.querySelector(`#${id}`);
```
Comment (erikhallaraaker) [inline src/px-script/main/utils/index.js]: If you are only getting elements by id's, why not use `document.getElementById`?

File: src/App/Documentation/components/Dialog/index.js (hunk 3 above)
Comment (Akswii) [reply src/App/Documentation/components/Dialog/index.js]: forgot to add it again 😄

**Pull request merged**